packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -264,4 +264,42 @@ void main() {
     expect(find.text('Alaska'), findsOneWidget);
   });
 
+  testWidgets('PageView peek', (WidgetTester tester) async {
```

Comment:
we should test that this handles the main axis extent changing (without rebuilding) when the peek is non-zero.